src/main/java/me/eccentric_nz/TARDIS/builders/TARDISInteriorPostioning.java

```diff
@@ -232,7 +232,7 @@ public void reclaimChunks(World w, int id) {
                             }
                             // remove dropped items
                             for (Entity e : tipsChunk.getEntities()) {
-                                e.remove();
+                                //e.remove();
```

Comment:
Are we not wanting to remove item drops that occur when blocks are removed eg torches dropping when the block they are attached is is removed? They will eventually despawn, but will cause some lag while they're still around? or do we not need to bother as they're removed when they fall into the void, I can never remember...
If we don't need to remove item entities then there's no point looping through them and all three lines can be removed.

Reply:
Ah, I think I forgot to uncomment that line, actually. Let me fix that and test real quick!